docs/01_nodeos/03_plugins/trace_api_plugin/index.md

```diff
@@ -101,23 +112,74 @@ nodeos --data-dir data_dir --config-dir config_dir --trace-dir traces_dir
 --trace-rpc-abi=eosio.wrap=abis/eosio.wrap.abi
 ```
 
-## Maintenance Note
+## Definitions
 
-To reduce the disk space consumed by the `trace_api_plugin`, configure the following option: 
+This section provides an overview of *slices*, the *trace log* contents, and the *clog format*. Mastery of these concepts is beneficial for an effective use of the `trace_api_plugin` options.
 
-```console
+### Slices
+
+In the context of the `trace_api_plugin`, a *slice* is defined as a collection of all relevant trace data between a given starting block height (inclusive) and a given ending block height (exclusive). For instance, a slice from 0 to 10,000 is a collection of all blocks with block numbers greater than or equal to 0 and less than 10,000. The trace directory contains a collection of slices. Each slice consists of a *trace data* log file and a *trace index* metadata log file:
+
+  *  `trace_<S>-<E>.log`
+  *  `trace_index_<S>-<E>.log`
+
+where `<S>` and `<E>` are the starting and ending block numbers for the slice padded with leading 0's to a stride. For instance if the start block is 5, the last is 15, and the stride is 4, then the resulting `<S>` is `0000000005` and `<E>` is `0000000015`.
```

Comment:
I think the TDD was incorrect.  The given examples are of stride `10` which is also what we use in the code.  Originally they correctly showed as 4 digits but we didn't sync the "stride" value when we true'd them to the code

Reply:
This is fixed now.